.github/workflows/tictactoe-daily-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Tictactoe Daily Deploy
+
+on:
+  push:
+    paths:
+      - sum/**
```

Comment:
shouldn't it be `games?/**`

Reply:
oops, copy/paste error ;)
thanks

done

Reply:
you're right. @lilyanB Can you fix it in an other branch